### ISSUE TITLE
bluetooth-hci-socket@0.5.3-8 install error question

### ISSUE THREAD

**Issue by caravan-icaner** [abandonware/node-bluetooth-hci-socket#42]
![image](https://user-images.githubusercontent.com/111466772/202583961-7c42cfa2-111c-4ba5-80ad-d4e74d666930.png)

While installing the Homebridge plug-in, it seems that the socket module cannot be installed as shown in the image above. What should I do?
I am using the ds218j model.

**Comment by Apollon77** [comment]
Yes then no build tools are installed on the host ... on linux you normally install "build-essential" package

**Comment by caravan-icaner** [comment]
Thanks for the answer.
Can you tell me how to install "build-essential" package on ds218j?
I want to know what command to enter into the putty program to install it

**Comment by Apollon77** [comment]
Honestly ... no idea ... I think "Google is your friend" :-)

**Comment by caravan-icaner** [comment]
thank you for answer !! ^^
